src/popupmessage.js

```diff
@@ -109,8 +115,9 @@ export default class PopUpMessage {
       document.activeElement.blur();
 
       // For works with Safari.
-      window.focus();
+      // window.focus();
```

Comment:
No comment code. BTW: is this code no longer necessary? Has been tried in Safari?